pyriemann_qiskit/classification.py

```diff
@@ -0,0 +1,296 @@
+                      device.configuration().n_qubits >= self._new_feature_dim
+                      and not device.configuration().simulator
+                      and device.status().operational)
+                devices = self._provider.backends(filters=filters)
+                try:
+                    self._backend = least_busy(devices)
+                except Exception:
+                    self._log("Devices are all busy. Getting the first one...")
+                    self._backend = devices[0]
+                self._log("Quantum backend = ", self._backend)
+            seed_sim = aqua_globals.random_seed
+            seed_trs = aqua_globals.random_seed
+            self._quantum_instance = QuantumInstance(self._backend, shots=1024,
+                                                     seed_simulator=seed_sim,
+                                                     seed_transpiler=seed_trs)
+        return self
+
+    def _run(self, predict_set=None):
+        raise Exception("Run method was not implemented")
+
+    def _self_calibration(self):
+        X = self._prev_fit_params["X"]
+        y = self._prev_fit_params["y"]
+        test_per = 0.33
+        self._log("Test size = ", test_per, " of previous fitting.")
+        X_train, X_test, y_train, y_test = train_test_split(X, y,
+                                                            test_size=test_per)
+        self.fit(X_train, y_train)
+        self.score(X_test, y_test)
+
+    def predict(self, X):
+        if(len(self.test_input) == 0):
+            self._log("There is no test inputs. Self-calibrating...")
+            self._self_calibration()
+        result = None
+        predict_set = self._vectorize(X)
+        self._log("Prediction: ", X.shape)
+        result = self._run(predict_set)
+        self._log("Prediction finished. Returning predicted labels")
+        return result["predicted_labels"]
+
+    def predict_proba(self, X):
+        self._log("""[WARNING] SVM prediction probabilities are not available.
+                 Results from predict will be used instead.""")
+        predicted_labels = self.predict(X)
+        ret = [np.array([c == 0, c == 1]) for c in predicted_labels]
+        return np.array(ret)
+
+    def score(self, X, y):
+        self._log("Scoring: ", X.shape)
+        VectorizedXta, VectorizedXnt = self._split_target_and_non_target(X, y)
+        self.test_input = {}
+        self.test_input["Target"] = VectorizedXta
+        self.test_input["NonTarget"] = VectorizedXnt
+        result = self._run()
+        balanced_accuracy = result["testing_accuracy"]
+        self._log("Balanced accuracy = ", balanced_accuracy)
+        return balanced_accuracy
+
+
+class QuanticSVM(QuanticBase):
+
+    """Quantum-enhanced SVM classification.
+
+    This class implements SVC on a quantum machine.
+    Note if `quantum` parameter is set to `False`
+    then a classical SVC will be perfomed instead.
+
+    See Also
+    --------
+    QuanticBase
+
+    """
```

Comment:
Can you add 1 or 2 references for QSVM?
```suggestion
    References
    ----------
    .. [1] TODO
    .. [2] TODO
    """
```

Reply:
Same remark for `VQC`.